### PR TITLE
Migrate evm dba

### PR DESCRIPTION
**Pull request by juliancheal**
Used my code extraction library to migrate over evm_dba.rake.

Fixes issue https://github.com/ManageIQ/manageiq-schema/issues/3

### CODE REVIEW

File: lib/tasks/evm_dba.rake [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+        opt :reindex,    "Reindex database (or table if --table specified)"
+        opt :analyze,    "Vacuum with analyze"
+        opt :full,       "Vacuum full"
+        opt :verbose,    "Vacuum with verbose information printed"
+
+        opt :table,      "Tablename to reindex (if only perorm on one)", :type => :string
+      end
+
+      opts = opts.delete_if { |_k, v| v.nil? || v == false }
+      EvmDatabaseOps.gc(opts)
```
Comment (Fryguy) [inline lib/tasks/evm_dba.rake]: EvmDatabaseOps should probably come over as well.

**Comment by Fryguy** [comment]
I generally like this idea, but want input from @bdunne / @jrafanie / @carbonin

File: lib/tasks/evm_dba.rake (hunk 1 above)
Comment (carbonin) [reply lib/tasks/evm_dba.rake]: There are a handful of places where we use models in that file. Are we okay with those being in this repo?

**Comment by jrafanie** [comment]
hmm, I like the idea of moving things where they belong but there's something weird about moving gc/backup/restore/postgresql stop/start/database configuration/ etc. to a schema repo.  They're only vaguely related in that they're for the same postgresql database.  For example, why not bring over postgresql HA stuff, our pg stat activity stuff and other PG related things here?  I can't say where they belong but it feels unrelated to schema.  Perhaps the wrapper for reset and migrate as they're more schema related but the others don't seem to fit.

**Pull request closed without merging**